flowey/flowey_lib_common/src/install_git.rs

```diff
@@ -39,9 +39,7 @@ impl FlowNode for Node {
         }
 
         let ensure_installed = ensure_installed;
-        let auto_install = auto_install.ok_or(anyhow::anyhow!(
-            "Missing essential request: LocalOnlyAutoInstall",
-        ))?;
+        let auto_install = auto_install.unwrap_or(false);
```

Comment:
Its a little boilerplate-ey at the moment, but look at other instances of `LocalOnly` requests to see how you should handle the different behaviors on each backend.

i.e: the request should still be required if you run on a local backend